input/haleu/inputs/united_states/reactors/nuscale_voygr.xml

```diff
@@ -7,12 +7,12 @@
         <fuel_inrecipes> <val>fresh_smr_fuel</val> </fuel_inrecipes>
         <fuel_outcommods> <val>spent_smr_fuel</val> </fuel_outcommods>
         <fuel_outrecipes> <val>spent_smr_fuel</val> </fuel_outrecipes>
-        <cycle_time>24</cycle_time>
```

Comment:
For future reference, you could include a comment or README with where you got the data for each reactor type from.

Reply:
I understand. Once I get another few code PRs in I plan to have one about documentation for this repo, because there is a lot that needs to be updated. I will make sure to include sources for reactor information.